src/routes/index.tsx

```diff
@@ -19,28 +19,21 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 import { AnimateSharedLayout } from 'framer-motion'
 import { Route, Routes } from 'react-router-dom'
 
-import UpdateWalletBanner from '@/components/UpdateWalletBanner'
-import { useGlobalContext } from '@/contexts/global'
 import HomePage from '@/pages/HomePage'
 
 import CreateWalletRoutes from './CreateWalletRoutes'
 import ImportWalletRoutes from './ImportWalletRoutes'
 import UnlockedWalletRoutes from './UnlockedWalletRoutes'
 
-const Router = () => {
-  const { newLatestVersion } = useGlobalContext()
-
-  return (
-    <AnimateSharedLayout type="crossfade">
-      {newLatestVersion && <UpdateWalletBanner newVersion={newLatestVersion} />}
-      <Routes>
-        <Route path="/create/:step" element={<CreateWalletRoutes />} />
-        <Route path="/import/:step" element={<ImportWalletRoutes />} />
-        <Route path="/wallet/*" element={<UnlockedWalletRoutes />} />
-        <Route path="" element={<HomePage />} />
-      </Routes>
-    </AnimateSharedLayout>
-  )
-}
+const Router = () => (
+  <AnimateSharedLayout type="crossfade">
```

Comment:
This has been removed as of framer motion 5. This layout refactor may be a good oppurtinity to upgrade! :)

Reply:
Nice, upgraded to Framer 5 and removed this!